src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/ChestSlimefunGuide.java

```diff
@@ -100,8 +101,10 @@ public boolean isSurvivalMode() {
         List<Category> categories = new LinkedList<>();
 
         for (Category category : SlimefunPlugin.getRegistry().getCategories()) {
-            if (!category.isHidden(p) && (!(category instanceof FlexCategory) || ((FlexCategory) category).isVisible(p, profile, getLayout()))) {
-                categories.add(category);
+            if ((!isSurvivalMode() && category instanceof SeasonalCategory) || !category.isHidden(p)) {
+                if (!(category instanceof FlexCategory) || ((FlexCategory) category).isVisible(p, profile, getLayout())) {
+                    categories.add(category);
+                }
             }
```

Comment:
I would probably not modify this method like that.
There is a dedicated class for the Cheat Sheet, so I reckon it may probably be wiser to override the method there (make it protected too of course) and then just display any non-flex category regardless of the visibility.

I think this design choice would make much more sense since someone might expect this to be in the cheat sheet class, as it is a special functionality of that particular object and not the normal guide.

Reply:
Okay, will do.